types/streamify-string/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-empty-interface": false
+  }
```

Comment:
sorry for missing this one, this should be inline comment, not change in config:
please do literallly as in README:
https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson
(there is a reason why folks here do not allow changes to tslint configs)